src/pages/home/ReportScreen.js

```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 import {Platform, View} from 'react-native';
 import lodashGet from 'lodash/get';
 import _ from 'underscore';
+import {Freeze} from 'react-freeze';
```

Comment:
@marcaaron why `react-freeze` isn't added to [package.json](https://github.com/Expensify/App/pull/11550/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) ?

Reply:
the dependency exists in other packages we are using so it's in node modules.